### DISCUSSION TITLE
How to turn off the default full-screen state when entering the editing mode?

### DISCUSSION THREAD

**Discussion by likaia** [hm21/pro_image_editor#261] [Q&A] [answered]
Hello, my component is placed in a container on the page. After I click Edit to enter the editing mode, it occupies the window of my entire application. How to turn off this default behavior? Let him only be within my container.

<img width="1250" alt="image" src="https://github.com/user-attachments/assets/b1c273f9-5c98-4ba4-b8c7-94ae5a1c4694">
<img width="1250" alt="image" src="https://github.com/user-attachments/assets/1141e0a6-0c1c-4a58-adcb-f745a73d501f">

**Comment by hm21** [comment, marked as answer]
To update that part, modify the `imageEditorTheme` property within the configuration settings as shown below. For more details, you can refer to the discuss [here](https://github.com/hm21/pro_image_editor/discussions/51).

```dart
configs: ProImageEditorConfigs(
      imageEditorTheme: ImageEditorTheme(
        subEditorPage: SubEditorPageTheme(
            enforceSizeFromMainEditor: true,
            positionTop: 0,
            positionLeft: 0,
            barrierDismissible: true,
            barrierColor: const Color(0x90272727),
            borderRadius: BorderRadius.circular(10),
            transitionsBuilder:
                (context, animation, secondaryAnimation, child) {
              return child;
            }),
      ),
```

**Reply by likaia** [reply]
It's done, thank you very much.
